content/widgets/answers.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Answers
+type: widget
+html: |
+  <div class="ais-Answers">
+    <div class="ais-Answers-header">Algolia Answers</div>
+    <ul class="ais-Answers-list">
+      <li class="ais-Answers-item">Author Sam Martin shares photos of ...</li>
+      <li class="ais-Answers-item">Beau Lotto's color games puzzle your vision, but they also spotlight what you can't normally see...</li>
+    </ul>
+  </div>
+alt1: empty
+althtml1: |
+  <div class="ais-Answers ais-Answers--empty">
+    <div class="ais-Answers-header">Algolia Answers</div>
+    <ul class="ais-Answers-list"></ul>
+  </div>
+alt2: loading
+althtml2: |
+  <div class="ais-Answers ais-Answers--empty">
+    <div class="ais-Answers-header">Algolia Answers</div>
+    <div class="ais-Answers-loader">loading...</div>
+    <ul class="ais-Answers-list">
+      <!-- Displays the hits from the last result -->
+      <li class="ais-Answers-item">Author Sam Martin shares photos of ...</li>
+      <li class="ais-Answers-item">Beau Lotto's color games puzzle your vision, but they also spotlight what you can't normally see...</li>
+    </ul>
```

Comment:
I'm surprised the old results still show when there's a new result loading, is this something you discussed with @dustincoates?

Reply:
@Haroenv thank you for bringing this up; I thought when I saw it that I had implemented the widget incorrectly. We want the results to disappear when a new query starts.